app-office/Focuswriter/focuswriter-1.6.7.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="A QT Word Processor"
+DESCRIPTION="Focuswriter is an Open Source QT Word Processor"
+HOMEPAGE="https://gottcode.org/focuswriter/"
+COPYRIGHT="2008-2017 Graeme Gott"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/portableapps/files/Source/FocusWriter/focuswriter-1.6.7-src.tar.bz2"
+CHECKSUM_SHA256="bf78e596d7ed9de774bef5bbe75f1f0a01d626198fce6c8062a679936dfc3883"
+SOURCE_DIR="focuswriter-$portVersion"
+ADDITIONAL_FILES="focuswriter.rdef.in"
+
+ARCHITECTURES="x86_64 ?x86 ?x86_gcc2"
+
+PROVIDES="
+	focuswriter$secondaryArchSuffix = $portVersion
+	app:Focuswriter$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libhunspell_1.6$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Script$secondaryArchSuffix
+	lib:libQt5Sql$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libhunspell_1.6$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Script$secondaryArchSuffix
+	devel:libQt5Sql$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix
+	cmd:sed
+	"
+
+BUILD()
+{
+	qmake FOCUSWRITER_PREFIX=$appsDir/Focuswriter
+	make$jobArgs
+}
+
+INSTALL()
+{	
+    mkdir -p $appsDir/Focuswriter   
```

Comment:
Single tab at the start of this line, no spaces. Also remove the trailing whitespace at the end.